### DISCUSSION TITLE
How to use custom commands

### DISCUSSION THREAD

**Discussion by venkat2305** [JunkFood02/Seal#438] [Q&A] [answered]
1. how to use yt-dlp commands in the seal app? Please give me an example.
2. Can you tell me which command to use to embed subtitles to audio files as live lyrics?

**Comment by Marco-9456** [comment, marked as answer]
1. Go to yt-dlp [USAGE AND OPTIONS](https://github.com/yt-dlp/yt-dlp#usage-and-options) there you'll find all the commands you can use in Seal `Custom Commands Template`
2. 
```
--downloader libaria2c.so
--external-downloader-args "aria2c:\"--summary-interval=1\""
--embed-thumbnail
--yes-playlist
--ppa "ffmpeg: -c:v png -vf crop=\"'if(gt(ih,iw),iw,ih)':'if(gt(iw,ih),ih,iw)'\""
--convert-subs lrc
--write-subs
--sub-langs en
-f ba
-x
--audio-quality 0
```

This Template downloads playlist/album's and thumbnail/cover-art crops it to look like a square then download the lyrics and convert it to lrc.
>**Note**
this will not embed subtitles as it looks like that yt-dlp doesn't embed lrc wit --embed-subs command

**Reply by venkat2305** [reply]
No lrc files were downloaded. What went wrong?

**Reply by Marco-9456** [reply]
Give a link

**Reply by venkat2305** [reply]
https://youtu.be/kTJczUoc26U

**Reply by Marco-9456** [reply]
> https://youtu.be/kTJczUoc26U

Add --write-subs

**Reply by venkat2305** [reply]
It worked. Thanks for your help.

**Reply by Marco-9456** [reply]
> It worked. Thanks for your help.

No problem, and please if you have a question always use the discussion section.

**Reply by venkat2305** [reply]
Should I request yt-dlp to add the ability to embed lrc files?

**Reply by Marco-9456** [reply]
> Should I request yt-dlp to add the ability to embed lrc files?

I didn't say it's impossible i said i just don't know how to do it, i think you should change the question to "how to embed lyrics to audio files ?"
